src/Illuminate/Foundation/Console/ModelMakeCommand.php

```diff
@@ -50,6 +50,22 @@ public function fire()
     }
 
     /**
+     * Ask if a new migration should be generated.
+     *
+     * @return bool
+     */
+    protected function confirmNewMigration()
+    {
+        $migrationClass = 'Create'.Str::plural((class_basename($this->argument('name')))).'Table';
+
+        if (class_exists($migrationClass)) {
+            return false;
+        }
+
+        return $this->confirm("A $migrationClass migration was not found. Do you want to generate it?", true);
```

Comment:
Please use curly braces around the variable name in the string. While it's not technically required in this case, it is Laravel's code standard to always put them around, for clarity.